### ISSUE TITLE
Remove acceptance tests in coding standard

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3422]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#3422|
|Type|defect|
|Created|2008-09-08 16:03:54Z|
|Branch|https://github.com/twisted/twisted/tree/acceptance-tests-3422|

There are no more acceptance tests and we should delete the part of the coding standard which discusses them.

<details><summary>Searchable metadata</summary>

```
trac-id__3422 3422
type__defect defect
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_acceptance_tests_3422 branches/acceptance-tests-3422
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1220889834000000 1220889834000000
changetime__1220912618000000 1220912618000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

(In [24733]) Branching to 'acceptance-tests-3422'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24748]) Merge acceptance-tests-3422: Remove references to acceptance tests in coding standard document.

Author: thijs
Reviewer: exarkun
Fixes: [#3422](https://github.com/twisted/twisted/issues/3422)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
